boards/cc2538dk/include/periph_conf.h

```diff
@@ -132,11 +132,11 @@ static const spi_clk_conf_t spi_clk_config[] = {
  */
 static const spi_conf_t spi_config[] = {
     {
-        .dev      = SSI0,
-        .mosi_pin = GPIO_PA4,
-        .miso_pin = GPIO_PA5,
-        .sck_pin  = GPIO_PA2,
-        .cs_pin   = GPIO_PD0
+        .num      = 0,
+        .mosi_pin = GPIO_PIN(0, 4),     /**< GPIO_PA4 */
+        .miso_pin = GPIO_PIN(0, 5),     /**< GPIO_PA5 */
+        .sck_pin  = GPIO_PIN(0, 2),     /**< GPIO_PA2 */
+        .cs_pin   = GPIO_PIN(3, 0)      /**< GPIO_PD0 */
```

Comment:
I know, this is it not the purpose of this PR. Anyway, my question is: is this a hardware CS line? If so, [`SPI_HWCS`](https://github.com/RIOT-OS/RIOT/blob/master/drivers/include/periph/spi.h#L98) needs to be handled defined in the CPU configuration.
Now that I had a deeper look at the SPI driver I seems it doesn't implement [`spi_init_cs`](https://github.com/RIOT-OS/RIOT/blob/master/drivers/include/periph/spi.h#L205). Can you fix that?!